test/widgets/inbox_test.dart

```diff
@@ -595,6 +607,38 @@ void main() {
           check(rectAfterTap).equals(rectBeforeTap);
         });
 
+        testWidgets('shows archived label for archived streams', (tester) async {
+          final stream = eg.stream(streamId: 1, isArchived: true);
```

Comment:
Looks like `streamId` being `1` is not required for this test. We can remove that to keep this test boring and focused.